[Security Solution] Allow prebuilt rules import and export

nikitaindik opened the pull request:
**Resolves: https://github.com/elastic/security-team/issues/11502** (internal)

This PR implements following changes and adds API integration tests for them:
- [x] Users with any license can export prebuilt rules (with enabled feature flag)
- [x] Users with Basic/Essentials license can import prebuilt rules only if they are non-customized and the feature flag is enabled
- [x] Users with Enterprise/Complete license can import prebuilt rules without restrictions

Flaky test runner (had to create 4 separate runs to test all configs):
 - [1](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7987)
 - [2](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7986)
 - [3](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7988)
 - [4](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7989)

<!--ONMERGE {"backportTargets":["8.18","8.x","9.0"]} ONMERGE-->

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/279410)
* Commit: ca3adf84d700e5ef33c92a2511375d1faab7f920

### Failed CI Steps
* [x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/customization_enabled/configs/serverless.config.ts](https://buildkite.com/elastic/kibana-pull-request/builds/279410#0195424e-e91c-4cb5-96f9-73da3c65d3d5)
* [x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/customization_enabled/configs/serverless.config.ts](https://buildkite.com/elastic/kibana-pull-request/builds/279410#01954252-ef40-46a4-bda2-49570892ffac)

cc @nikitaindik

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279410","number":279410,"commit":"ca3adf84d700e5ef33c92a2511375d1faab7f920"}],"number":279410}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/280284)
* Commit: 8df69e509fa4dacb5fed0c9fddb1ec5586e9f670

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/280284#01955bf2-6cc8-478d-a3ba-0660b5340f4c)

### History
* :broken_heart: [Build #279410](https://buildkite.com/elastic/kibana-pull-request/builds/279410) failed ca3adf84d700e5ef33c92a2511375d1faab7f920

cc @nikitaindik

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280284","number":280284,"commit":"8df69e509fa4dacb5fed0c9fddb1ec5586e9f670"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279410","number":279410,"commit":"ca3adf84d700e5ef33c92a2511375d1faab7f920"}],"number":280284}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/280321)
* Commit: d3e1af5f4edfc79e413960f1b81bc0734d078f2a

### Failed CI Steps
* [FTR Configs #11](https://buildkite.com/elastic/kibana-pull-request/builds/280321#01955c5e-ce5c-4e37-9990-83e6b89e99cf)
* [FTR Configs #67](https://buildkite.com/elastic/kibana-pull-request/builds/280321#01955c5e-cea2-42a7-99ea-d6428f6fa7c8)
* [FTR Configs #67](https://buildkite.com/elastic/kibana-pull-request/builds/280321#01955c82-c6da-4193-af18-9c65a4be8682)

### History
* :broken_heart: [Build #280284](https://buildkite.com/elastic/kibana-pull-request/builds/280284) failed 8df69e509fa4dacb5fed0c9fddb1ec5586e9f670
* :broken_heart: [Build #279410](https://buildkite.com/elastic/kibana-pull-request/builds/279410) failed ca3adf84d700e5ef33c92a2511375d1faab7f920

cc @nikitaindik

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280321","number":280321,"commit":"d3e1af5f4edfc79e413960f1b81bc0734d078f2a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280284","number":280284,"commit":"8df69e509fa4dacb5fed0c9fddb1ec5586e9f670"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279410","number":279410,"commit":"ca3adf84d700e5ef33c92a2511375d1faab7f920"}],"number":280321}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/280675)
* Commit: 08b9f1bf6974c6789cd0fb977313f5f15df76774

### Failed CI Steps
* [Osquery Cypress Tests on Serverless #6](https://buildkite.com/elastic/kibana-pull-request/builds/280675#019560da-6870-4a47-bc8d-16faedef09b9)

### History
* :broken_heart: [Build #280321](https://buildkite.com/elastic/kibana-pull-request/builds/280321) failed d3e1af5f4edfc79e413960f1b81bc0734d078f2a
* :broken_heart: [Build #280284](https://buildkite.com/elastic/kibana-pull-request/builds/280284) failed 8df69e509fa4dacb5fed0c9fddb1ec5586e9f670
* :broken_heart: [Build #279410](https://buildkite.com/elastic/kibana-pull-request/builds/279410) failed ca3adf84d700e5ef33c92a2511375d1faab7f920

cc @nikitaindik

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280675","number":280675,"commit":"08b9f1bf6974c6789cd0fb977313f5f15df76774"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280321","number":280321,"commit":"d3e1af5f4edfc79e413960f1b81bc0734d078f2a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280284","number":280284,"commit":"8df69e509fa4dacb5fed0c9fddb1ec5586e9f670"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279410","number":279410,"commit":"ca3adf84d700e5ef33c92a2511375d1faab7f920"}],"number":280675}
buildkite-pr-comment-->

elasticmachine commented:
Pinging @elastic/security-solution (Team: SecuritySolution)

elasticmachine commented:
Pinging @elastic/security-detection-rule-management (Team:Detection Rule Management)

jbudz reviewed (approved):
`.ftr_configs.yml` LGTM

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/280813)
* Commit: 828f36ce0ded0a2cc22a9d12a6c5cf6e1f6b9509

### Failed CI Steps
* [Osquery Cypress Tests #7](https://buildkite.com/elastic/kibana-pull-request/builds/280813#019561de-c24e-4be4-84f0-d4cefa2e8502)

### History
* :broken_heart: [Build #280675](https://buildkite.com/elastic/kibana-pull-request/builds/280675) failed 08b9f1bf6974c6789cd0fb977313f5f15df76774
* :broken_heart: [Build #280321](https://buildkite.com/elastic/kibana-pull-request/builds/280321) failed d3e1af5f4edfc79e413960f1b81bc0734d078f2a
* :broken_heart: [Build #280284](https://buildkite.com/elastic/kibana-pull-request/builds/280284) failed 8df69e509fa4dacb5fed0c9fddb1ec5586e9f670
* :broken_heart: [Build #279410](https://buildkite.com/elastic/kibana-pull-request/builds/279410) failed ca3adf84d700e5ef33c92a2511375d1faab7f920

cc @nikitaindik

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280813","number":280813,"commit":"828f36ce0ded0a2cc22a9d12a6c5cf6e1f6b9509"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280675","number":280675,"commit":"08b9f1bf6974c6789cd0fb977313f5f15df76774"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280321","number":280321,"commit":"d3e1af5f4edfc79e413960f1b81bc0734d078f2a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280284","number":280284,"commit":"8df69e509fa4dacb5fed0c9fddb1ec5586e9f670"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279410","number":279410,"commit":"ca3adf84d700e5ef33c92a2511375d1faab7f920"}],"number":280813}
buildkite-pr-comment-->

banderror commented:
@nikitaindik Can you please run all the new and changed tests via the flaky test runner?

xcrzx commented on x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/export/bulk_action_export.ts:
Can we move this test to `rule_import_export/export_prebuilt_rules/bulk_action_export.ts`? It would be nice to have all export-related tests in one place. This will also reduce the number of configs we need to manage, as the conditions for bulk export and regular export APIs are the same.

xcrzx commented on x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/export/bulk_action_export.ts:
```diff
@@ -0,0 +1,63 @@
+      await deleteAllPrebuiltRuleAssets(es, log);
+    });
+
+    it(`exports prebuilt rules if the feature flag is enabled`, async () => {
```
```suggestion
    it(`exports prebuilt rules`, async () => {
```

xcrzx commented on x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/import_customized_prebuilt_rules/feature_disabled/not_allowed_importing_customized_prebuilt_rules.ts:
```diff
@@ -0,0 +1,62 @@
+      await deleteAllPrebuiltRuleAssets(es, log);
+    });
+
+    it(`does NOT import customized prebuilt rules when feature flag is disabled`, async () => {
```
```suggestion
    it(`does NOT import customized prebuilt rules when rule customization is disabled`, async () => {
```

nikitaindik commented on x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/trial_license_complete_tier/import_rules.ts:
```diff
@@ -1626,6 +1626,7 @@ export default ({ getService }: FtrProviderContext): void => {
     describe('supporting prebuilt rule customization', () => {
       describe('compatibility with prebuilt rule fields', () => {
         it('rejects rules with "immutable: true" when the feature flag is disabled', async () => {
+          // duplicate test
```
Heh, good catch! This was a note to myself that I forgot to handle :)

xcrzx reviewed (changes requested):
Tested the changes locally and found a few issues with Export buttons disabled in UI

**1. FF enabled, sufficient license**

- 🟢 Prebuilt rules can be exported
- 🟢 Customized rules can be imported
- 🟢 Non-customized rules can be imported

**2. FF enabled, insufficient license**

- 🔴 Prebuilt rules cannot be exported, the export option is disabled. It should be enabled
  <img width="287" alt="image" src="https://github.com/user-attachments/assets/d606a0fd-76f6-4da4-8096-c6544ca8b127" />
- 🟡 Customized rules cannot be imported. This is expected, but the error message should be more clear saying that the rule cannot be imported because it is **Modified** (we don't use the term customized in UI)
  <img width="626" alt="image" src="https://github.com/user-attachments/assets/85cdb883-0238-4901-be0a-5102c52b73c4" />
- 🟢 Non-customized rules can be imported

**3. FF disabled, sufficient license**
- 🔴 Prebuilt rules cannot be exported, the export option is disabled. It should be enabled
  <img width="287" alt="image" src="https://github.com/user-attachments/assets/d606a0fd-76f6-4da4-8096-c6544ca8b127" />
- 🟢 Customized rules cannot be imported
- 🟢 Non-customized rules cannot be imported
  <img width="623" alt="image" src="https://github.com/user-attachments/assets/6e646841-e271-4669-ac94-8241ff32eee2" />

nikitaindik commented:
Thanks for review and testing, @xcrzx! As discussed I've reverted the exporting behaviour to only allow exporting if the feature flag is disabled.

Also, I've addressed your feedback, so feel free to do another review pass.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/281155)
* Commit: 502bf2992b3ea6716fc93b8c097c59b34f0e3ad8

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/281155#01956701-af81-482f-86b6-bc8152f038c8)

### History
* :broken_heart: [Build #280813](https://buildkite.com/elastic/kibana-pull-request/builds/280813) failed 828f36ce0ded0a2cc22a9d12a6c5cf6e1f6b9509
* :broken_heart: [Build #280675](https://buildkite.com/elastic/kibana-pull-request/builds/280675) failed 08b9f1bf6974c6789cd0fb977313f5f15df76774
* :broken_heart: [Build #280321](https://buildkite.com/elastic/kibana-pull-request/builds/280321) failed d3e1af5f4edfc79e413960f1b81bc0734d078f2a
* :broken_heart: [Build #280284](https://buildkite.com/elastic/kibana-pull-request/builds/280284) failed 8df69e509fa4dacb5fed0c9fddb1ec5586e9f670
* :broken_heart: [Build #279410](https://buildkite.com/elastic/kibana-pull-request/builds/279410) failed ca3adf84d700e5ef33c92a2511375d1faab7f920

cc @nikitaindik

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281155","number":281155,"commit":"502bf2992b3ea6716fc93b8c097c59b34f0e3ad8"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280813","number":280813,"commit":"828f36ce0ded0a2cc22a9d12a6c5cf6e1f6b9509"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280675","number":280675,"commit":"08b9f1bf6974c6789cd0fb977313f5f15df76774"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280321","number":280321,"commit":"d3e1af5f4edfc79e413960f1b81bc0734d078f2a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280284","number":280284,"commit":"8df69e509fa4dacb5fed0c9fddb1ec5586e9f670"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279410","number":279410,"commit":"ca3adf84d700e5ef33c92a2511375d1faab7f920"}],"number":281155}
buildkite-pr-comment-->

kibanamachine commented:
## Flaky Test Runner Stats
### 🎉 All tests passed! - [kibana-flaky-test-suite-runner#7986](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7986)
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/import_non_customized_prebuilt_rules/feature_enabled/configs/ess_basic_license.config.ts: 50/50 tests passed.
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/import_non_customized_prebuilt_rules/feature_disabled/configs/ess_feature_flag_disabled.config.ts: 50/50 tests passed.
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/import_customized_prebuilt_rules/feature_enabled/configs/ess_enterprise_license.config.ts: 50/50 tests passed.
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/import_customized_prebuilt_rules/feature_disabled/configs/ess_basic_license.config.ts: 50/50 tests passed.

[see run history](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds?branch=refs%2Fpull%2F212509%2Fhead)

kibanamachine commented:
## Flaky Test Runner Stats
### 🟠 Some tests failed. - [kibana-flaky-test-suite-runner#7984](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7984)
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/import_non_customized_prebuilt_rules/feature_enabled/configs/serverless_essentials_tier.config.ts: 50/50 tests passed.
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/import_non_customized_prebuilt_rules/feature_disabled/configs/serverless_feature_flag_disabled.config.ts: 50/50 tests passed.
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/import_customized_prebuilt_rules/feature_enabled/configs/serverless_complete_tier.config.ts: 50/50 tests passed.
[❌] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/import_customized_prebuilt_rules/feature_disabled/configs/serverless_essentials_tier.config.ts: 28/50 tests passed.

[see run history](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds?branch=refs%2Fpull%2F212509%2Fhead)

xcrzx reviewed (approved):
Retested locally, and all previously tested scenarios are now working as expected 👍  

Leaving a minor comment regarding test scripts - looks like they’re pointing to the wrong test files after refactoring. I also don’t see much value in maintaining these test scripts. There are a lot of them, making navigation difficult. Using the FTR server and runner directly seems much more practical. So, I’d suggest removing the added test scripts from `package.json` altogether.

nikitaindik commented on x-pack/test/security_solution_api_integration/package.json:
```diff
@@ -457,6 +492,16 @@
     "rule_bulk_actions:runner:ess": "npm run run-tests:rm rule_bulk_actions ess essEnv",
 
+    "rule_bulk_actions_export:customization_disabled:server:ess": "node ./scripts/index.js server detections_response/rules_management customization_disabled rule_bulk_actions/export ess",
+    "rule_bulk_actions_export:customization_disabled:runner:ess": "node ./scripts/index.js runner detections_response/rules_management customization_disabled rule_bulk_actions/export ess essEnv",
```
Removed them altogether 👍

kibanamachine commented:
## Flaky Test Runner Stats
### 🎉 All tests passed! - [kibana-flaky-test-suite-runner#7988](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7988)
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/import_customized_prebuilt_rules/feature_disabled/configs/serverless_feature_flag_disabled.config.ts: 50/50 tests passed.
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/export_prebuilt_rules/feature_enabled/configs/serverless_essentials_tier.config.ts: 50/50 tests passed.
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/export_prebuilt_rules/feature_disabled/configs/serverless_feature_flag_disabled.config.ts: 50/50 tests passed.

[see run history](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds?branch=refs%2Fpull%2F212509%2Fhead)

kibanamachine commented:
## Flaky Test Runner Stats
### 🎉 All tests passed! - [kibana-flaky-test-suite-runner#7989](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7989)
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/import_customized_prebuilt_rules/feature_disabled/configs/serverless_essentials_tier.config.ts: 50/50 tests passed.

[see run history](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds?branch=refs%2Fpull%2F212509%2Fhead)

kibanamachine commented:
Starting backport for target branches: 8.18, 8.x, 9.0

https://github.com/elastic/kibana/actions/runs/13702844293

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/281431)
* Commit: a76914ac776a62ea09c93ece491ed8bfd05e32de

### Failed CI Steps
* [Jest Tests #8](https://buildkite.com/elastic/kibana-pull-request/builds/281431#01956bd2-1d0e-4d03-a010-e588cbd8afe5)
* [FTR Configs #24](https://buildkite.com/elastic/kibana-pull-request/builds/281431#01956bd2-1d28-4299-9aa6-50178b1cf44a)
* [FTR Configs #111](https://buildkite.com/elastic/kibana-pull-request/builds/281431#01956bd2-1d6b-4633-a6e7-921068c2acb5)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/281431#01956bd2-1d6b-4633-a6e7-921068c2acb5) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/281431/jobs/01956bd2-1d6b-4633-a6e7-921068c2acb5/artifacts/01956bef-0b0c-45b3-94fc-092accb94cb8) FTR Configs #<span></span>111 / Stateful Observability - Deployment-agnostic API integration tests SyntheticsAPITests getSyntheticsMonitors get many monitors without params
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/281431#01956bd2-1d0e-4d03-a010-e588cbd8afe5) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/281431/jobs/01956bd2-1d0e-4d03-a010-e588cbd8afe5/artifacts/01956bfe-a025-4b0d-912f-4d6a16391df4) Jest Tests #<span></span>8 / Tags it changes the tags
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/281431#01956bd2-1d0e-4d03-a010-e588cbd8afe5) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/281431/jobs/01956bd2-1d0e-4d03-a010-e588cbd8afe5/artifacts/01956bfe-a026-4776-ad63-d15e0388a7ce) Jest Tests #<span></span>8 / Tags it shows error when tag is empty

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/5010d1f0fa52e28085f03fb1f6e03004586eaab1) | [after](https://github.com/elastic/kibana/commit/a76914ac776a62ea09c93ece491ed8bfd05e32de) | diff |
| --- | --- | --- | --- |
| `securitySolution` | 8.9MB | 8.9MB | +10.0B |



### History
* :broken_heart: [Build #281370](https://buildkite.com/elastic/kibana-pull-request/builds/281370) failed 70ff48c4e3d50efcb22e36b9a13ba7811ca3b0aa
* :broken_heart: [Build #281155](https://buildkite.com/elastic/kibana-pull-request/builds/281155) failed 502bf2992b3ea6716fc93b8c097c59b34f0e3ad8
* :broken_heart: [Build #280813](https://buildkite.com/elastic/kibana-pull-request/builds/280813) failed 828f36ce0ded0a2cc22a9d12a6c5cf6e1f6b9509
* :broken_heart: [Build #280675](https://buildkite.com/elastic/kibana-pull-request/builds/280675) failed 08b9f1bf6974c6789cd0fb977313f5f15df76774
* :broken_heart: [Build #280321](https://buildkite.com/elastic/kibana-pull-request/builds/280321) failed d3e1af5f4edfc79e413960f1b81bc0734d078f2a

cc @nikitaindik

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281431","number":281431,"commit":"a76914ac776a62ea09c93ece491ed8bfd05e32de"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281370","number":281370,"commit":"70ff48c4e3d50efcb22e36b9a13ba7811ca3b0aa"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281155","number":281155,"commit":"502bf2992b3ea6716fc93b8c097c59b34f0e3ad8"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280813","number":280813,"commit":"828f36ce0ded0a2cc22a9d12a6c5cf6e1f6b9509"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280675","number":280675,"commit":"08b9f1bf6974c6789cd0fb977313f5f15df76774"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280321","number":280321,"commit":"d3e1af5f4edfc79e413960f1b81bc0734d078f2a"}],"number":281431}
buildkite-pr-comment-->

kibanamachine commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.18|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/213419">](https://github.com/elastic/kibana/pull/213419)|
|✅|8.x|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/213420">](https://github.com/elastic/kibana/pull/213420)|
|✅|9.0|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/213421">](https://github.com/elastic/kibana/pull/213421)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->

pborgonovi commented:
/ci

pborgonovi commented:
/ci

pborgonovi commented:
buildkite test this
